recipes/link_traits/meta.yaml

```diff
@@ -0,0 +1,43 @@
+package:
+  name: link-traits
+  version: "1.0.1"
+
+source:
+  fn: link_traits-1.0.1.tar.gz
+  url: https://pypi.python.org/packages/64/d1/4d683994fbba2d3691f7433c09453ea8f4a5144a74767bd39c0d0f8ad25a/link_traits-1.0.1.tar.gz
```

Comment:
Please use the canonical pypi.io link as in:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18